src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,93 @@
+package duke;
+
+import duke.command.CommandBuilder;
+import duke.exception.DukeException;
+import duke.exception.DukeSideEffectException;
+import duke.io.FileIO;
+import duke.io.Printer;
+import duke.task.Task;
+import duke.task.TaskList;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Duke {
+
+  private final String BOT_NAME = "GOAT";
+  private final String FILEPATH = "data/";
+  private final String SAVEFILE_NAME = "save.txt";
```

Comment:
Consistent naming conventions are followed. However, it's better to follow the Java naming convention of having constant variables (BOT_NAME, FILEPATH, etc.) as static final